overlord/snapstate/snapmgr.go

```diff
@@ -644,26 +645,37 @@ func (m *SnapManager) localInstallCleanup() error {
 
 // Ensure implements StateManager.Ensure.
 func (m *SnapManager) Ensure() error {
+	perftimings := timings.New(map[string]string{"ensure": "snap manager"})
+	measure := func(f func() error, label, summary string) error {
+		meas := perftimings.StartSpan(label, summary)
+		err := f()
+		meas.Stop()
+		return err
+	}
+
 	// do not exit right away on error
 	errs := []error{
 		m.atSeed(),
-		m.ensureAliasesV2(),
+		measure(m.ensureAliasesV2, "aliases", ""),
```

Comment:
I don't  think measuring at this granularity is that useful, also it creates the problem that we get very short times most of the time, I would  undo the changes here for this PR and we can discuss what to do about Ensures separately.

Reply:
Done.